geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/mapreduce/AvroFileInputFormat.scala

```diff
@@ -19,7 +19,7 @@ import org.opengis.feature.simple.SimpleFeature
 
 /**
  * Class for reading avro files written using
- * <code>org.locationtech.geomesa.features.avro.AvroDataFileWriter</code>.
+ * <code>org.locationtech.geomesa.features.avro.AvroDataFileReader</code>.
```

Comment:
@dalbani I think we are using the AvroDataFileWriter to create the files.

Perhaps the comment can be made more clear in another way?